<launch>
    <rosparam command = "load" file = "$(find platforms_server)/config/vision.config" />
    <rosparam command = "load" file = "$(find platforms_server)/config/robot.config" />
    <rosparam command = "load" file = "$(find platforms_server)/config/mqtt.config" />
    <rosparam command = "load" file = "$(find platforms_server)/config/path_planner.config" />

    <node pkg="platforms_server" name="aruco_detector_node" type="aruco_detector_node.py" output="screen"/>
    <node pkg="platforms_server" name="camera_node" type="camera_node.py" output="screen"/>
    <node pkg="platforms_server" name="markers_analizer_node" type="markers_analizer_node.py" output="screen"/>
    <node pkg="platforms_server" name="visualizer_node" type="visualizer_node.py" output="screen"/>
    <node pkg="platforms_server" name="test_set_path" type="test_set_path.py" output="screen"/>
    <node pkg="platforms_server" name="mqtt_node" type="mqtt_node.py" output="screen"/>

    <node pkg="platforms_server" name="parallel_connection_node" type="parallel_connection_node.py" output="screen"/>

    <node pkg="platforms_server" name="fine_tune_node" type="fine_tune_node.py" output="screen"/>

</launch>
